.sops.yaml

```diff
@@ -0,0 +1,37 @@
+keys:
+  - &mic92 age17n64ahe3wesh8l8lj0zylf4nljdmqn28hvqns2g7hgm9mdkhlsvsjuvkxz
```

Comment:
This can be either a pgp fingerprint or an age key.

Reply:
For gpg: https://github.com/Mic92/sops-nix#2a-generate-a-gpg-key-for-yourself
For age: https://github.com/Mic92/sops-nix#2a-generate-a-ssh-and-age-key-for-yourself

Reply:
Oh, is there not support for using SSH keys directly?

Reply:
No the upstream Age maintainer did not want this when reviewing the sops PR that added age support. However we build an easy to use tool that does the conversion.